google/cloud/asset_v1/types/assets.py

```diff
@@ -19,12 +19,14 @@
 
 
 from google.cloud.orgpolicy.v1 import orgpolicy_pb2 as orgpolicy  # type: ignore
+from google.cloud.osconfig.v1 import inventory_pb2 as inventory  # type: ignore
```

Comment:
Asset is depending on a type in a different API.

https://github.com/googleapis/python-os-config

Dependency on OS Config added in: https://github.com/googleapis/googleapis/commit/1f8a5144b52f7677dc43c55b21ccaf9e1b425ceb

Plan: 
- regenerate google-cloud-os-config
- add regex replacements in this file to use `google.cloud.osconfig_v1.Inventory` 

Does that sound correct @software-dov?